packages/patterns/src/keys/checkKey.js

```diff
@@ -415,7 +415,7 @@ export const getCopyMapEntries = m => {
     payload: { keys, values },
   } = m;
   const { length } = /** @type {Array} */ (keys);
-  return Far('CopyMap entries iterable', {
+  return harden({
```

Comment:
TODO test in agoric-sdk.

Not sure why `getCopyMapEntries` alone returns an Iterable whereas its siblings all return arrays. Probably that is what should really be fixed, though hopefully not in this PR.

We make the returned Iterable non-passable. Hopefully that is consistent with all uses in agoric-sdk. Otherwise, hopefully all uses are compat with this returning an array, which would be passable.

Reply:
Done. Changes to returning an array, and removed the other name. Nothing was sensitive to whether this returns an only-Iterable vs an array which also happens to be an Iterable. The other name was never exported from this package, so could be safely removed. Done.